customer.go

```diff
@@ -22,6 +22,7 @@ type Customer struct {
 	AndroidPayCards    *AndroidPayCards          `xml:"android-pay-cards,omitempty"`
 	ApplePayCards      *ApplePayCards            `xml:"apple-pay-cards,omitempty"`
 	PaymentMethodNonce string                    `xml:"payment-method-nonce,omitempty"`
+	Addresses          *Addresses                `xml:"addresses,omitempty"`
```

Comment:
We can't really add this here because `Customer` is used for both requests and responses and it isn't valid to pass addresses when creating a customer. That has to be done separately through the address gateway.